update.ps1

```diff
@@ -328,33 +332,23 @@ try {
             break
         }
     }
+}
+else {
+    $previousAccount = $account
+}
 
-    # Define ExportData with account fields and correlation property 
-    $exportData = $account.PsObject.Copy() | Select-Object $storeAccountFields
-    # Add correlation property to exportdata
-    $exportData | Add-Member -MemberType NoteProperty -Name $correlationProperty -Value $correlationValue -Force
-    # Add aRef properties to exportdata
-    foreach ($aRefProperty in $aRef.PSObject.Properties) {
-        $exportData | Add-Member -MemberType NoteProperty -Name $aRefProperty.Name -Value $aRefProperty.Value -Force
-    }
+# Check if auditLogs contains errors, if no errors are found, set success to true
+if (-NOT($outputContext.AuditLogs.IsError -contains $true)) {
```

Comment:
because at the start of the script $outputContext.Success = $true is set, even if the auditlogs contains IsError $true, then the $outputContext.Success is not set to $false?

Reply:
Copy error. resolved in new commit